releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201907_2019.07.29.txt

```diff
@@ -0,0 +1,183 @@
+Starting build on  Mon Jul 29 21:40:13 UTC 2019
+     + sudo mv /var/log /var/log.vhd
+Using kernel:
+Linux version 4.15.0-1052-azure (buildd@lgw01-amd64-001) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.10)) #57-Ubuntu SMP Tue Jul 23 19:07:16 UTC 2019
+Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
+  - apt-transport-https
+  - auditd
+  - blobfuse
+  - ca-certificates
+  - ceph-common
+  - cgroup-lite
+  - cifs-utils
+  - conntrack
+  - cracklib-runtime
+  - ebtables
+  - ethtool
+  - fuse
+  - git
+  - glusterfs-client
+  - init-system-helpers
+  - iproute2
+  - ipset
+  - iptables
+  - jq
+  - libpam-pwquality
+  - libpwquality-tools
+  - mount
+  - nfs-common
+  - pigz socat
+  - util-linux
+  - xz-utils
+  - zip
+  - etcd v3.2.26
+  - moby v3.0.6
+  - nvidia-docker2 nvidia-container-runtime
+  - Azure CNI version 1.0.24
+  - Azure CNI version 1.0.22
+  - CNI plugin version 0.7.5
+  - CNI plugin version 0.7.1
+  - containerd version 1.2.4
+  - containerd version 1.1.6
+  - containerd version 1.1.5
+  - img
+Docker images pre-pulled:
+     + sudo mkdir /var/log
+     + applyCIS
+  - k8s.gcr.io/kubernetes-dashboard-amd64:v1.10.1
+  - k8s.gcr.io/exechealthz-amd64:1.2
+     + setPWExpiration
+  - k8s.gcr.io/addon-resizer:1.8.5
+     + sed -i 's|PASS_MAX_DAYS||g' /etc/login.defs
+     + grep PASS_MAX_DAYS /etc/login.defs
+  - k8s.gcr.io/addon-resizer:1.8.4
+  - k8s.gcr.io/addon-resizer:1.8.1
+     + sed -i 's|PASS_MIN_DAYS||g' /etc/login.defs
+  - k8s.gcr.io/addon-resizer:1.7
+  - k8s.gcr.io/heapster-amd64:v1.5.4
+  - k8s.gcr.io/heapster-amd64:v1.5.3
+  - k8s.gcr.io/heapster-amd64:v1.5.1
+     + grep PASS_MIN_DAYS /etc/login.defs
+  - k8s.gcr.io/metrics-server-amd64:v0.2.1
+     + sed -i 's|INACTIVE=||g' /etc/default/useradd
+     + grep INACTIVE= /etc/default/useradd
```

Comment:
This shouldn't be in the release notes, looks like the script didn't generate notes properly

Reply:
Manually removed